7_react/react1/week2/todoList2/src/components/todoList/TodoList.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import TodoItem from './TodoItem'; 
+import BorderComponent from './BorderComponents';
+
+function TodoList() {
+    const [todos, setTodos] = useState([]);
+    const [newTodoDescription, setNewTodoDescription] = useState('');
+    const [newTodoDeadline, setNewTodoDeadline] = useState('');
```

Comment:
well done with the date time picker! This is just suggestion, but feel free to use it if you want. 

1. You can define the current date by something like this
```
const today = new Date().toISOString().split("T")[0];
function TodoList() {
 const [newTodoDeadline, setNewTodoDeadline] = useState(today);
```
Thanks to that the current day will be always as default value
2. set the current value as `min` date for the input
```
 <input
                type="date"
                min={today}
                value={newTodoDeadline}
```
Thanks to this, you do not need to check whether the selected date is in the past, because the picker won't let you pick the date :) I think this could improve the User experience a bit :)